### DISCUSSION TITLE
Differences in the clone info between exportClones and exportAirr

### DISCUSSION THREAD

**Discussion by ShaowenJ** [milaboratory/mixcr#1436] [Q&A] [answered]
Dear Mixcr developers,

Sorry for this naive question, but I realized the output from the exportAirr files, that in the "cdr3_aa" column the first letter was missing compared to "junction_aa" column.

Is that a reason for this?
Example:
<img width="920" alt="image" src="https://github.com/milaboratory/mixcr/assets/30546732/448d367b-3f61-4a8a-ab70-ba38cb7f01e7">

Thanks,
Shaowen

**Comment by mizraelson** [comment, marked as answer]
Hi, in compliance with AIRR format [description](https://docs.airr-community.org/en/stable/datarep/rearrangements.html):

"  Junction versus CDR3
We work with the IMGT definitions of the junction and CDR3 regions. Specifically, the IMGT JUNCTION includes the conserved cysteine and tryptophan/phenylalanine residues, while CDR3 excludes those two residues. Therefore, our junction and junction_aa fields which represent the extracted sequence include the two conserved residues, while the coordinate fields (cdr3_start and cdr3_end) exclude them."
